lib/Web2Print/Processor.php

```diff
@@ -90,11 +90,13 @@ public function preparePdfGeneration($documentId, $config)
 
         if (!$disableBackgroundExecution) {
             $process->start();
-            $logHandle = fopen(PIMCORE_LOG_DIRECTORY . DIRECTORY_SEPARATOR . 'web2print-output.log', 'a');
-            $process->wait(function ($type, $buffer) use ($logHandle) {
-                fwrite($logHandle, $buffer);
-            });
-            fclose($logHandle);
+            //we can't write output to log file, since wait() is a blocking method
```

Comment:
@dvesh3 @fashxp Don't think this is a problem so we can remove all the comments. But one thing: 
https://github.com/pimcore/pimcore/blob/dd9fc1a906b7b6eed72af8ebffa9733baf1f9708/bundles/CoreBundle/Command/Web2PrintPdfCreationCommand.php#L53 
This should be logged to the standard log, so that this message isn't getting lost.

Reply:
@dvesh3  can you please revert all these changes, we will solve that with https://github.com/pimcore/pimcore/pull/8371

Reply:
we can close this PR and copy over upgrade notes to #8371

Reply:
Done